jetson/src/core/robot.cpp

```diff
@@ -153,7 +156,39 @@ void Robot::moveServoArm(ServoMotor motor, double pos) {
 	servoArm[motor].setIdealSpeed(0);
 }
 
-// TODO: perform pollination pattern
-void Robot::pollinate() { }
+void Robot::pollinate() { 
+	double delta = 2.5;
+	Point3D currPos = getArmPosition();
+	moveServoArm(x, currPos.x - delta);
+	moveServoArm(y, currPos.y - delta);
+	moveServoArm(x, currPos.x + delta);
+	moveServoArm(y, currPos.y + delta);
+	moveServoArm(x, currPos.x);
+	moveServoArm(y, currPos.y);
+}
+
+bool debug = false;
+std::vector<Point3D> Robot::scan() {
+	std::vector<Point3D> flowersToVisit;
+	for (int i = 1; i < 4; i++){
+		for (int j = 1; j < 3; j++){
+			moveServoArm(x, robotWidth*i/4);
+			moveServoArm(y, robotLength*j/3);
```

Comment:
this won't really be robot dimensions. Refer to cartesian min/maxs in `core/dimensions.h`.